src/fmgc/src/guidance/vnav/FlightModel.ts

```diff
@@ -52,25 +52,25 @@ export class FlightModel {
         let baseDrag;
         switch (flapConf) {
         case FlapConf.CLEAN:
-            baseDrag = (-0.1043 * Cl ** 5) + (0.2635 * Cl ** 4) - (0.2319 * Cl ** 3) + (0.1537 * Cl ** 2) - (0.0379 * Cl) + 0.0233;
+            baseDrag = (0.0211 * Cl ** 3) + (0.0412 * Cl ** 2) - (0.015 * Cl) + 0.0215;
             break;
         case FlapConf.CONF_1:
-            baseDrag = (-0.0207 * Cl ** 5) + (0.0764 * Cl ** 4) - (0.0813 * Cl ** 3) + (0.0912 * Cl ** 2) - (0.0285 * Cl) + 0.0337;
+            baseDrag = (0.0385 * Cl ** 3) + (0.004 * Cl ** 2) + (0.0044 * Cl) + 0.0249;
             break;
         case FlapConf.CONF_2:
-            baseDrag = (0.0066 * Cl ** 5) - (0.0271 * Cl ** 4) + (0.0615 * Cl ** 3) - (0.0187 * Cl ** 2) + (0.0035 * Cl) + 0.0538;
+            baseDrag = (0.0242 * Cl ** 3) + (0.0013 * Cl ** 2) + (0.003 * Cl) + 0.0485;
             break;
         case FlapConf.CONF_3:
-            baseDrag = (0.0768 * Cl ** 5) - (0.3979 * Cl ** 4) + (0.8252 * Cl ** 3) - (0.7951 * Cl ** 2) + (0.3851 * Cl) - 0.0107;
+            baseDrag = (0.0221 * Cl ** 3) - (0.0027 * Cl ** 2) + (0.0033 * Cl) - 0.0584;
```

Comment:
Hi, did a quick test with VNAV and predictions were off. Is it possible this minus sign is incorrect? The curve is very similar to the old one if the sign is switched,
```suggestion
            baseDrag = (0.0221 * Cl ** 3) - (0.0027 * Cl ** 2) + (0.0033 * Cl) + 0.0584;
```